pkg/nsx/services/vpc/builder.go

```diff
@@ -89,3 +93,17 @@ func buildNSXVPC(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, nc common.VPCNe
 
 	return vpc, nil
 }
+
+func buildNSXLBS(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, cluster, lbsSize, vpcPath string, relaxScaleValidation *bool) (*model.LBService, error) {
+	lbs := &model.LBService{}
+	lbsName := util.GenerateDisplayName("", "vpc", nsObj.GetName(), "", cluster)
+	// LBS id should equal VPC id
+	lbs.Id = common.String(string(nsObj.GetUID()))
+	lbs.DisplayName = &lbsName
+	// TODO(gran) do we need "created_for" and "lb_t1_link_ip" tag?
```

Comment:
@dantingl Here is the missing tags. I think `lb_t1_link_ip` is not available for this case.

Reply:
`lb_t1_link_ip` is not needed. `created_for` should be added.

Reply:
No need to expose lb_t1_link_ip for now.

Reply:
Fixed this part.